utils/structured-data.ts

```diff
@@ -0,0 +1,33 @@
+import { h, toDisplayString } from 'vue'
+import type { MaybeRef } from 'vue'
+
+type structuredDataContent = Record<string, unknown>
+
+const isNullishValue = (v: unknown) => {
+    if (typeof v === 'number') return false
+    return v === null || v === undefined || !Object.keys(v).length || (Array.isArray(v) && !v.length)
+}
```

Comment:
I think it should be split into another util file, e.g. `/util/null.ts` (group with other utils for nullish values) or `/util/is-nullish-value.ts`